pages/data/products/ozonesonde.vue

```diff
@@ -1,87 +1,369 @@
 <template>
-  <products-page
-    :title="$t('title')"
-    :instructions="$t('instructions')"
-    dataset="OzoneSonde"
-    hide-instruments
-  />
+  <v-layout justify-center column align-content-center>
```

Comment:
`<v-layout>` is deprecated in Vuetify, in favour of `v-container` and `v-row`/`v-col`: https://vuetifyjs.com/en/components/grids/#api 

I just noticed that `@nuxtjs/vuetify` is on version 1.0.0, which certainly doesn't help the case. I think you'll need to do:
- `npm update` 
- `npm install`
- `npm audit fix` (to clean up some vulnerabilities)

I just ran those and started up `npm run dev` to work nicely. There were some warnings to be fixed; mainly due to `<i18n>` blocks and the old/unused components that need to be removed.

Reply:
After running those steps and doing `npm list vuetify` I get
```
 -- @nuxtjs.vuetify@1.11.2
   -- vuetify@2.3.1
```

No new warnings or errors show up, nothing about `v-layout` (which is used in a lot of places) or any `<i18n>` blocks. Is that still an old version?